GitOut/Features/Navigation/NavigationService.cs

```diff
@@ -74,12 +72,12 @@ public void Back()
             {
                 return;
             }
-            (ContentControl current, string _, IServiceScope scope) = pageStack.Pop();
-            (ContentControl latest, string? title, IServiceScope _) = pageStack.Peek();
+            (ContentControl current, string _) = pageStack.Pop();
+            (ContentControl latest, string? title) = pageStack.Peek();
             OnNavigationRequested(latest);
             titleService.Title = title;
             CurrentPage = latest.GetType().FullName;
-            logger.LogInformation(LogEventId.Navigation, $"Navigating back to {CurrentPage} ({title})");
+            logger.LogInformation(LogEventId.Navigation, "Navigating back to {CurrentPage} ({Title})", CurrentPage, title);
```

Comment:
```suggestion
            logger.LogInformation(LogEventId.Navigation, $"Navigating back to {CurrentPage} ({Title})", CurrentPage, title);
```

Reply:
Apparently not: https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2254
As I understand it, the reason is two-fold, the logger formats the string anyway by using the attached properties, and by using a constant log string and passing the values as items, the logger can use the names in the log format string to name the variables passed to it, in Application Insights these are then bound to a property bag called `customDimensions` that you can query specifically.